src/data/auth/projectServices.ts

```diff
@@ -0,0 +1,37 @@
+import {
+  ProjectsData,
+  NewProject,
+  DeleteProject,
+  Project,
+} from '../../types/project';
+import { http } from '../http/http';
+
+export const getAllProjects = async (): Promise<ProjectsData> => {
+  const { data } = await http.get<ProjectsData>('/projects');
+  return data;
+};
+
+export const getProjectById = async (id: string): Promise<Project> => {
+  const { data } = await http.get<Project>(`/projects/${id}`);
+  return data;
```

Comment:
Will React Query handle errors in case, if you return only data from request?

Reply:
On the example of auth.services, I think React Query will handle errors in case